Mathlib/Init/Data/Nat/Lemmas.lean

```diff
@@ -209,20 +213,24 @@ protected theorem one_le_bit0 : ∀ n : ℕ, n ≠ 0 → 1 ≤ bit0 n
 end bit
 
 /- successor and predecessor -/
-
+/--`discriminate` is basically the principle of mathematical induction-/
```

Comment:
If this comment wasn't on the original lemma, then it doesn't belong in a porting PR. This comment didn't exist in mathlib3.

Reply:
Thus came up with #lint along with the other doc strings requests.

Reply:
There are already ignores in this file for those lint errors, you don't need to fix them just because `#lint` finds them

Reply:
Okay I will remove them.

Reply:
Done. Removed all the doc strings that I added, from that file.